installscripts/jazz-terraform-unix-noinstances/acl.tf

```diff
@@ -0,0 +1,43 @@
+resource "aws_rds_cluster" "casbin" {
+  cluster_identifier      = "casbin-cluster"
+  availability_zones      = ["us-east-1a", "us-east-1b"]
+  database_name           = "casbin"
+  master_username         = "root"
+  master_password         = "casbin2715"
+  backup_retention_period = 5
+  preferred_backup_window = "07:00-09:00"
+  skip_final_snapshot = true
+  engine                  = "aurora-mysql"
```

Comment:
i dont see the mysql version (5.6, 5.7). Are you going with default?

Reply:
"engine". https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/AuroraMySQL.Updates.20180206.html#AuroraMySQL.Updates.20180206.CLI

Reply:
Updated the version as well.